doc/source/math_num_documentation/forward_structure.rst

```diff
@@ -496,6 +496,133 @@ Hydrological processes can be described at pixel scale in `smash` with one of th
         q_t(x, t) = q_r(x, t) + q_{e}(x, t) + q_d(x, t) 
 
 
+
+.. _math_num_documentation.forward_structure.hydrological_module.grc:
+
+
+.. dropdown:: grc (Génie Rural c)
+    :animate: fade-in-slide-down
+
+    This hydrological operator is derived from the GR6 model. It consists in a gr6 like model stucture (see diagram above) but with a slow-tansfert reservoir instead of the exponential reservoir. This structure is used for the Reunion island.
+
+    .. hint::
+
+        Helpful links about GR:
+
+        - `Brief history of GR models <https://webgr.inrae.fr/models/a-brief-history/>`__
+        - `Scientific papers <https://webgr.inrae.fr/publications/articles/>`__
+        - `GR models in a R package <https://hydrogr.github.io/airGR/>`__
+
+    .. figure:: ../_static/gr6_structure.svg
+        :align: center
+        :width: 400
+        
+        Diagram of the ``grc`` like hydrological operator
+
+    It can be expressed as follows:
+
+    .. math::
+
+        q_{t}(x, t) = f\left(\left[P, E\right](x, t), m_{lt}(x, t), \left[c_i, c_p, c_t, c_l, k_{exc}, a_{exc}\right](x), \left[h_i, h_p, h_t, h_l\right](x, t)\right)
+
+    with :math:`q_{t}` the elemental discharge, :math:`P` the precipitation, :math:`E` the potential evapotranspiration,
+    :math:`m_{lt}` the melt flux from the snow module, :math:`c_i` the maximum capacity of the interception reservoir,
+    :math:`c_p` the maximum capacity of the production reservoir, :math:`c_t` the maximum capacity of the transfer reservoir,
+    :math:`c_l` the maximum capacity of the slow-transfer reservoir, 
+    :math:`k_{exc}` the exchange coefficient, :math:`a_{exc}` the exchange threshold, :math:`h_i` the state of the interception reservoir, 
+    :math:`h_p` the state of the production reservoir and :math:`h_t` the state of the transfer reservoir,
+    :math:`h_l` the state of the slow transfer reservoir.
```

Comment:
h_t and h_l not in agreement with notation on diagram (update diagram since this is h_t l in code below)